src/ring_theory/ideal/norm.lean

```diff
@@ -410,8 +419,26 @@ lemma prime_of_irreducible_abs_norm_span {a : S} (ha : a ≠ 0)
   prime a :=
 (ideal.span_singleton_prime ha).mp (is_prime_of_irreducible_abs_norm hI)
 
+lemma quotient.index_eq_zero {S : Type*} [comm_ring S] (I : ideal S) :
```

Comment:
This can go to a more basic file. It it is already ported and you prefer to leave it here you can add a note.

Reply:
I think this is the first (relevant) file that has both `ideal.quotient` and `add_subgroup.index`. So I'll make a new file importing both.

Reply:
Aha, `algebra.char_p.quotient` should be a good target actually.

Reply:
If `find_home.py` works any file in the following list should be ok
```
{'mathlib:ring_theory.ideal.over', 'mathlib:field_theory.normal', 'mathlib:field_theory.separable', 'mathlib:ring_theory.localization.integral', 'mathlib:ring_theory.adjoin.fg', 'mathlib:algebraic_geometry.prime_spectrum.noetherian', 'mathlib:linear_algebra.matrix.adjugate', 'mathlib:ring_theory.power_basis', 'mathlib:ring_theory.ideal.local_ring', 'mathlib:linear_algebra.matrix.charpoly.basic', 'mathlib:group_theory.specific_groups.cyclic', 'mathlib:ring_theory.polynomial.rational_root', 'mathlib:field_theory.is_alg_closed.algebraic_closure', 'mathlib:linear_algebra.matrix.charpoly.minpoly', 'mathlib:ring_theory.jacobson_ideal', 'mathlib:field_theory.is_alg_closed.basic', 'mathlib:field_theory.minpoly.basic', 'mathlib:data.zmod.basic', 'mathlib:algebraic_geometry.prime_spectrum.maximal', 'mathlib:group_theory.perm.fin', 'mathlib:ring_theory.dedekind_domain.ideal', 'mathlib:ring_theory.adjoin.field', 'mathlib:linear_algebra.matrix.charpoly.linear_map', 'mathlib:ring_theory.integral_closure', 'mathlib:ring_theory.fractional_ideal', 'mathlib:field_theory.intermediate_field', 'mathlib:linear_algebra.matrix.polynomial', 'mathlib:linear_algebra.matrix.reindex', 'mathlib:linear_algebra.matrix.mv_polynomial', 'mathlib:linear_algebra.matrix.to_linear_equiv', 'mathlib:group_theory.perm.cycle.type', 'mathlib:field_theory.primitive_element', 'mathlib:field_theory.perfect_closure', 'mathlib:field_theory.galois', 'mathlib:field_theory.minpoly.field', 'mathlib:ring_theory.localization.away', 'mathlib:linear_algebra.matrix.nondegenerate', 'mathlib:ring_theory.localization.norm', 'mathlib:ring_theory.integral_domain', 'mathlib:data.nat.totient', 'mathlib:ring_theory.finite_presentation', 'mathlib:group_theory.exponent', 'mathlib:ring_theory.adjoin_root', 'mathlib:ring_theory.adjoin.tower', 'mathlib:data.zmod.quotient', 'mathlib:ring_theory.localization.at_prime', 'mathlib:ring_theory.dedekind_domain.basic', 'mathlib:linear_algebra.matrix.basis', 'mathlib:algebra.char_p.two', 'mathlib:ring_theory.localization.as_subring', 'mathlib:linear_algebra.free_module.ideal_quotient', 'mathlib:ring_theory.finite_type', 'mathlib:linear_algebra.matrix.determinant', 'mathlib:ring_theory.polynomial.basic', 'mathlib:linear_algebra.free_module.determinant', 'mathlib:ring_theory.algebraic', 'mathlib:ring_theory.norm', 'mathlib:field_theory.fixed', 'mathlib:linear_algebra.matrix.charpoly.coeff', 'mathlib:ring_theory.localization.localization_localization', 'mathlib:linear_algebra.determinant', 'mathlib:field_theory.adjoin', 'mathlib:algebra.char_p.basic', 'mathlib:linear_algebra.matrix.nonsingular_inverse', 'mathlib:data.polynomial.expand', 'mathlib:field_theory.splitting_field', 'mathlib:algebraic_geometry.prime_spectrum.basic', 'mathlib:ring_theory.integrally_closed'}
```

Reply:
Oops, forgot to push. I still think `algebra.char_p.quotient` is a good home :)